Clinical-Genomics/cgbeacon2
# Pybedtools functions to merge gene panels

How to test this:
- Remove all eventual variants from variant collection (and clear dataset sample list)
- Filter for variants in genomic intervals contained in 2 gene panels: 
`cgbeacon2 add variants -ds test_ds -vcf cgbeacon2/resources/demo/643594.clinical.vcf.gz -sample ADM1059A1 -panel /Users/chiararasi/Documents/work/GITs/cgbeacon2/cgbeacon2/resources/demo/panel1.bed -panel /Users/chiararasi/Documents/work/GITs/cgbeacon2/cgbeacon2/resources/demo/panel2.bed `
- Only variants in these regions should be uploaded
